src/component/Switch.jsx

```diff
@@ -0,0 +1,329 @@
+import React from "react";
+import styled from "styled-components";
+
+const Switch = () => {
+  const setDarkMode = () => {
+    document.querySelector("body").setAttribute("data-theme", "dark");
+    localStorage.setItem("selectedTheme", "dark");
+  };
+  const setlightMode = () => {
+    document.querySelector("body").setAttribute("data-theme", "light");
+    localStorage.setItem("selectedTheme", "light");
+  };
+
+  const selectedTheme = localStorage.getItem("selectedTheme");
+
+  if (selectedTheme === "dark") {
```

Comment:
Ovo nije dobra praksa. Kada god se desi neka promena ( bilokakva ) ovaj kod ce se izvrsiti bez razloga. Ovo treba da se desava samo ako se promeni vrednost. Ako ti je problem to sto localstorage nije reaktivan, pogledaj ovo https://www.geeksforgeeks.org/reactjs-uselocalstorage-custom-hook/ ili potrazi neki drugi resurs. Ovo sad nije kriticno jer ne postoji nesto sto ce opteretiti procesor i memoriju ali treba da budes svestan da se ovo desava stalno i bez razloga